### ISSUE TITLE
Can't disable manual shortcut for "Duplicate entry" function.

### ISSUE THREAD

**Issue by amweber** [willsALMANJ/Zutilo#31]
I exprimented with local shortcuts for Zutilo function via the Zutilo prefs interface. Under "Duplicate entry" (I'm using a German localisation and FF interface) I added "spacebar" as the shortcut – an admittedly daft idea. However, trying to disable the shortcut I find that I cannot do so. What works with every other entry (add/disable personal shortcuts via the interface) does not work for duplicate entry. (The the shortcut slot for Duplicate entry neither shows "disabled" nor what the spacebar would produce in any other field, which is "VK_space". A reinstall of the add-on doesn't help – my preferences including the buggy one are preserved somewehre. I'm using ZoteroQuicklook and its spacebar functions many times a day – and now duplicate all my viewed entries... Had to deactivate Zutilo to stop this from happening but would love to keep it's very useful funtionality. Any ideas how to fix this? Delete my prefs and start anew? Thanks

**Comment by willsALMANJ** [comment]
Hopefully, reseting just the one pref value will fix the problem. Try opening `about:config` in your browser (and agree to be careful in the screen that appears after that if you haven't opened `about:config` before) and enter part of `extensions.zutilo.shortcut.duplicateItem` into the search bar (or just `extensions.zutilo` if you want to see all of the preferences). Then right-click on the `extensions.zutilo.shortcut.duplicateItem` entry and choose "Reset". If Zutilo was enabled at any time since you last restarted Firefox, this should reset the value to `{"modifiers":"","key":"","keycode":""}`; otherwise, it will reset the value to be empty.

Before you reset the preference, note the value that it has (you can right click and "Copy value"). I'm curious to know what key or keycode setting it has. The behavior you see is different from what I see. When I tried this, I got `VK_space` for the "Duplicate item" shortcut as expected, so I am not sure how the setting was corrupted for you. Also, I can't get `VK_space` to do anything for me by itself (pressing spacebar still goes through to Firefox and scrolls the current page down rather than duplicating an item). I didn't try with the ZoteroQuickLook add-on. Perhaps it modifies the spacebar input somehow. Also, German keyboards have some different keycodes from English ones, though I would think spacebar would still be the same and so that wouldn't matter in this case.

**Comment by amweber** [comment]
Thanks for your prompt reply. I fixed my problem – but to quickly so to provide you with the desired key-string. I followed along your suggestion while reading your mail, slightly embarrassed I didn’t think of clearing the settings in about:config <about:config> myself and by the time I arrived at your last paragraph requesting the omnious string in the settings, the latter was already gone. I hope to reproduce the issue by using spacebar again as shortcut, however I couldn’t replicate the behavior I’d encountered earlier today. (I have the visual memory of a string with something weird in there, quite likely from my clipboard – I’m using Asian languages and scripts). If I try a simple spacebar I get exactly the behavior you describe. Thanks for your help and sorry for not being able to satisfy your curiosity.
